source/specifications.rst

```diff
@@ -9,11 +9,58 @@ toolchain, the Python Packaging Authority is also responsible for maintaining
 the interoperability specifications used to define the interactions between
 those tools.
 
-Specification Process
+Active Specifications
 ---------------------
 
-PyPA interoperability specifications are currently handled as Informational
-Python Enhancement Proposals in accordance with :pep:`1`.
+The currently active specifications are recorded in the
+:ref:`PyPA Specifications <pypug:specifications>` section of the
+Python Packaging User Guide.
+
+
+Specification Update Process
+----------------------------
+
+PyPA interoperability specifications are separated into two categories:
+
+* `Package Distribution Metadata <https://packaging.python.org/specifications/#package-distribution-metadata>`_
+* `Package Index Interfaces <https://packaging.python.org/specifications/#package-index-interfaces>`_
+
+For Package Distribution Metadata, the default responsible decision maker is
+the lead CPython representative on distutils-sig.
+
+For Package Index Interfaces, the default responsible decision maker is
+the lead maintainer for the `Python Package Index <https://pypi.org>`__.
+
+The current lead PyPI maintainer is Donald Stufft, while the current lead
+CPython representative on distutils-sig is Nick Coghlan.
+
+
+Handling fixes and other minor updates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Backwards compatible fixes and small enhancements to the specifications are
+handled by submitting pull requests to the specifications section of the guide.
```

Comment:
Good point, the actual process we switched to (at least for now) was to keep the Informational PEPs, but allow for minor in-place updates as along as a change log is kept in the PEP itself.

Moving the specifications into PyPUG wholesale was an alternative I'd proposed in #11, but we've never actually pursued. Since this PR is about bringing the description in line with what we *currently* do, I'll change both the part you mention here as well as the section on maintaining the informational PEPs.

I also noticed we never updated the list of active PEPs to include PEP 518.